### PR TITLE
Made same modifications made to fh_interface repo to create workshop …

### PR DESCRIPTION
**Pull request by JamesAndrews3**
…scene

Tested in simulation

### CODE REVIEW

**Comment by JamesAndrews3** [comment]
Tested in simulation - works as expected
NOT tested on real Hand E and UR10 in workshop!

File: sr_robot_launch/launch/sr_ur_arm_hand.launch [hunk 1]
```diff
@@ -125,10 +143,10 @@
     </include>
     <include file="$(find sr_multi_moveit_config)/launch/default_warehouse_db.launch"/>
 
-    <node pkg="sr_multi_moveit_config" type="timed_roslaunch.sh" args="10 sr_robot_launch trajectory_services.launch
-      arm_group_name:=$(arg arm_group_name)" name="timed_trajectory_services" output="screen"
-    />
-
+    <!-- Set scene for robot after delay to start after moveit - restrains movement -->
+    <node if="$(arg scene)" pkg="fh_robot_launch" type="timed_roslaunch.sh" args="8 sr_moveit_hand_config scene.launch
```
Comment (toliver) [inline sr_robot_launch/launch/sr_ur_arm_hand.launch]: There can't be a dependency on fh_robot_launch inside sr_interface.

There must be a copy of the script timed_roslaunch.sh somewhere in this repository I think.
Maybe ask Beatriz where.

**Review by toliver — CHANGES_REQUESTED** [review]

File: sr_moveit_hand_config/launch/scene.launch [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<launch>
+  <!-- This launch file sets the scene for robot - restrains movement -->
+
+  <arg name="scene_file" default="$(find fh_description)/other/scenes/lab.scene"/>
```
Comment (toliver) [inline sr_moveit_hand_config/launch/scene.launch]: Again, no dependencies on fh_... packages are allowed in sr_interface.

**Review by toliver — CHANGES_REQUESTED** [review]

**Review by toliver — APPROVED** [review]

File: sr_robot_launch/scripts/timed_roslaunch.sh [hunk 3]
```diff
@@ -0,0 +1,44 @@
+#!/bin/bash          
+#
```
Comment (beatrizleon) [inline sr_robot_launch/scripts/timed_roslaunch.sh:2]: This timed_roslaunch.sh is already in sr_interface in https://github.com/shadow-robot/sr_interface/tree/kinetic-devel/sr_multi_moveit/sr_multi_moveit_config/scripts so it doesn't need to be included again

File: sr_moveit_hand_config/launch/scene.launch [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<launch>
```
Comment (beatrizleon) [inline sr_moveit_hand_config/launch/scene.launch:1]: This launch file should be better placed inside the moveit config of the hand and arm rather that in the hand config only as it is meant to be used when both are present. In my opinion it should be placed here: https://github.com/shadow-robot/sr_interface/tree/kinetic-devel/sr_multi_moveit/sr_multi_moveit_config/launch

**Comment by toliver** [comment]
@beatrizleon Sorry I merged before I saw your comments. I have added your comments to a subtask to be done in this sprint.

**Pull request merged**